idb/postgres/postgres_random_box_test.go

```diff
@@ -0,0 +1,225 @@
+package postgres
```

Comment:
I separated this out from `postgres_random_test.go` because of all the merge issues. But I can unify those scripts again. LMK